.github/workflows/format.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       - name: Install formatters
         run: |
           python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools wheel
           pip install autoflake black isort
```

Comment:
You can combine these, or at least choose between `python3 -m pip` vs `pip`

Reply:
[Github](https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#requirements-filel) recommends splitting up these steps:
```bash
python -m pip install --upgrade pip
pip install -r requirements.txt
```
I think `python3` is redundant compared to `python` because `actions/setup-python@v1` takes care of putting the correct version in the PATH (the containers come with many versions preinstalled).

Reply:
splitting up pip upgrade and install is fine but why are setuptools and wheel separate from autoflake, black, isort

Reply:
> why are setuptools and wheel separate from autoflake, black, isort
Because the updating of pip, setuptools, and wheel is a shared set of common tasks to do when setting up a new python environment, and I want to seperate the shared tasks from the unique tasks.

Maybe it would make sense to split up this step into two steps.